gym/lib/gym/xcodebuild_fixes/swift_fix.rb

```diff
@@ -27,7 +27,15 @@ def swift_library_fix
           ipa_swift_frameworks.each do |path|
             framework = File.basename(path)
 
-            FileUtils.copy_file("#{Xcode.xcode_path}/Toolchains/XcodeDefault.xctoolchain/usr/lib/swift/iphoneos/#{framework}", File.join(swift_support, framework))
+            begin
+              from = File.join(Xcode.xcode_path, "Toolchains/XcodeDefault.xctoolchain/usr/lib/swift/iphoneos/#{framework}")
```

Comment:
@KrauseFx, this is to make it work with the legacy build api and xcode8 with Swift_2_3 toolchain. 
We can grab Gym.config[:toolchain] value and replace it with XcodeDefailt.xctoolchain. 

Something like this - 
```
if Gym.config[:toolchain]
	from = File.join(Xcode.xcode_path, "Toolchains/#{Gym.config[:toolchain}/usr/lib/swift/iphoneos/#{framework}")
else
	from = File.join(Xcode.xcode_path, "Toolchains/XcodeDefault.xctoolchain/usr/lib/swift/iphoneos/#{framework}")
end
```

This will make sure that legacy_build_api version will still work with xcode8 and swift_2_3 toolchain.